cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,79 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
+  const userName = 'iravakula';
+  const myPassword = 'Ira@12345';
+
+  const book = {
+    title: 'Speaking JavaScript',
+    desription: 'Like it or not, JavaScript is everywhere these days'
+  };
+
+  const alertMessage = {
+    added: 'Book added to your collection.',
+    deleted: 'Book deleted.'
+  };
+
   before(() => {
+    cy.visit('https://demoqa.com/login');
+    cy.viewport(1920, 1080);
+  });
+
+  it('should successfully loggin the user with valid credentials', () => {
+    cy.getByPlaceholder('UserName').type(userName);
+
+    cy.getByPlaceholder('Password').type(myPassword);
+
+    cy.get('#login').click();
+
+    cy.url().should('contain', 'profile');
+
+    cy.get('#userName-value').should('exist').and('contain', userName);
+  });
+  it('should provide ability to add book to the Cart', () => {
+    cy.login(userName, myPassword);
```

Comment:
The custom command 'cy.login()' is used but not defined anywhere in the code. It should be defined in the commands file.